designs/virtualization/4.5/Scheduling/Scheduling-section/Scheduling-section.md

```diff
@@ -0,0 +1,18 @@
+---
+parent: Virtualization
+version: 4.5
+---
+
+# Scheduling Section
+
+Creating a new section in the VM details page, Overview tab
+('Scheduling': If, where and how this workload will be placed within the cluster's nodes)
+
+![Operator details page](img/vm-page-0-0.jpg)
+
+Introducing a new section within the Details tab: `Scheduling and resources requirements`.
+This section comes right after the Virtual Machine Details section.
+This section contains all the properties that are set from this workload and affect this workload's scheduling.
+The two columns are sorted so the left column is dedicated for label-based kubernetes scheduling policies.
+The right column is dedicated for resources request and management.
+Each field within this section will be discussed in a different PR.
```

Comment:
```suggestion

```

Reply:
I think we can drop this from our documentation since it won't be needed once merged. In the future, you could add something like this to the PR details.